src/components/input-field/input-field.tsx

```diff
@@ -68,6 +68,12 @@ export class InputField {
     @Prop({ reflectToAttr: true })
     public trailingIcon: string;
 
+    /**
+     * Trailing icon to show to the far right in the field
+     */
+    @Prop({ reflectToAttr: true })
+    public trailingErrorIcon: string;
```

Comment:
I don't think we need the ability to customize the error icon, the default one should be enough for now. If there is demand for a custom icon we can add it later.